src/Shipping/ShippingZone.php

```diff
@@ -378,4 +452,26 @@ public static function is_shipping_method_valid( string $method ): bool {
 			true
 		);
 	}
+
+	/**
+	 * Finds and returns the free shipping method if it exists in the list of suggested shipping methods.
+	 *
+	 * @param array $methods
+	 *
+	 * @return array|null Array containing the free shipping method properties, or null if it does not exist.
+	 */
+	protected static function find_free_shipping_method( array $methods ): ?array {
+		$free_shipping_method = array_filter(
+			$methods,
+			function ( $method ) {
+				return self::METHOD_FREE === $method['id'];
```

Comment:
❓ 

I think we need to add `$method['enable'] === true` otherwise it will return a $free_shipping_method even when is disabled.

Reply:
Yes good catch. Added it in https://github.com/woocommerce/google-listings-and-ads/pull/1192/commits/796623752b289c71a8145343e8d40acb8674579b

Reply:
Thanks! 👍